parm/config/config.resources.emc.dyn

```diff
@@ -302,9 +302,10 @@ elif [ $step = "eobs" -o $step = "eomg" ]; then
     elif [ $CASE = "C96" -o $CASE = "C48" ]; then
       export npe_eobs=14
     fi
-    export nth_eobs=7
+    export nth_eobs=3
     if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_eobs=7; fi
     export npe_node_eobs=$(echo "$npe_node_max / $nth_eobs" | bc)
+    if [[ "$machine" = "WCOSS2" ]]; then export npe_node_eobs=40; fi
```

Comment:
Why is this override here? The math works out to 40 anyway, doesn't it?

Reply:
If I don't force it to be 40 the setup script math sets it to 42 (128/3=42.6). Here is the xml diff if I remove that line and rerun setup_workflow.py:
```
Kate.Friedman@dlogin03:/lfs/h2/emc/global/noscrub/Kate.Friedman/expdir/testcyc768o> diff testcyc768o.xml testcyc768o.xml_23
13c13
<               This workflow was automatically generated at 2022-01-31 20:10:45.310360
---
>               This workflow was automatically generated at 2022-01-31 16:22:16.683725
143c143
<       <!ENTITY RESOURCES_EOBS_GDAS "<nodes>12:ppn=42:tpp=3</nodes>">
---
>       <!ENTITY RESOURCES_EOBS_GDAS "<nodes>12:ppn=40:tpp=3</nodes>">
```

Reply:
`npe_node_max` is 128 on WCOSS2.   128 / 3 = 42.66667.   This doesn't round to 40.

Reply:
Ah, I thought it was 120. Sorry.